practice/timezone/timezone/views.py

```diff
@@ -0,0 +1,57 @@
+"""timezone Views."""
+from . import logic
+import arrow
+
+from django.http import HttpResponse
+
+
+def return_local_time(response):
+    local_time_string = logic.get_time_at_tz('US/Pacific')
+    return HttpResponse('Local time is: ' + local_time_string)
+
+
+def return_tz_at_latlng(response, latlng):
+    try:
+        tz_at_latlng = logic.get_tz(latlng)
+    except IOError as error:
+        return HttpResponse(error, status=400)
+    except KeyError:
+        return HttpResponse('Time Zone does not exist there.', status=404)
+    response_string = 'TimeZone at {} is {}'.format(latlng, tz_at_latlng)
+    return HttpResponse(response_string)
+
+
+def return_time_at_latlng(response, latlng):
+    try:
+        time_at_latlng = logic.get_time_at_latlng(latlng)
+    except IOError as error:
+        return HttpResponse(error, status=400)
+    except KeyError:
+        return HttpResponse('Time Zone does not exist there.', status=400)
+    response_string = 'Time at {} is {}'.format(latlng, time_at_latlng)
+    return HttpResponse(response_string)
+
+
+def return_converted_time(response, in_time, in_latlng, out_latlng):
```

Comment:
I fixed the problem definition for this part a day into the problem since before it didn't really make logical sense. I told everyone that in hot tips during that class period. It will also get rid of some of the more confusing logic functions.